src/docs/patterns/user-auth.mdx

```diff
@@ -0,0 +1,296 @@
+    ]}
+/>
+
+Your user authentication workflow should work on different screen sizes. On mobile, it should take up an entire screen. On a desktop, it is usually "boxed" in a card. The content should follow the rules specified by our [forms and validation pattern](/patterns/forms), and each page should be connected using our [steppers pattern](/patterns/steppers).
+
+<MaterialDesignDescription 
+    title={'Forms and Validation'}
+    url={'/patterns/forms'}
+    description={`Follow PX Blue's guideline for form controls.`}
+     avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+/>
+<MaterialDesignDescription 
+    title={'Steppers'}
+    url={'/patterns/steppers'}
+    description={`Follow PX Blue's guideline for navigating between pages.`}
+     avatar={<PxblueSmallAltIcon style={{ width: 48, height: 48 }} />}
+/>
+
+Currently, PX Blue supports the following authentication workflows:
+
+-   Registration
+-   Join organization
+-   Login
+-   Reset password
+
+We will now discuss them one by one.
+
+## Registration
+
+Registration is the process of user account creation. Depends on your project setup, the user might need to request to join various organizations, which we will discuss later. Registration includes the following variations:
+
+**Self-Registration**
+
+Self-registration involves registering an account without an email invitation. The workflow involves the following screens:
+
+-   Enter email address
+-   Accept EULA
+-   Verify email
+-   Create password
+-   Add account details
+-   Success & welcome
+
+**Registration via Invitation**
+
+Sometimes the organization admin might email the user an invitation. The workflow involves the following screens:
+
+-   Click through an email invitation
+-   Accept EULA
+-   Create password
+-   Add account details
+-   Success & welcome
+
+### Screens
+
+Most steps involved in self-registration and registration via invitation are very similar. Use the following screens for the registration workflow where appropriate.
+
+<ImageGrid
+    images={[EnterEmailAdress, AcceptEULA, VerifyEmail, CreatePassword, AccountDetails, WelcomeOnboard]}
+    caption={[
+        'Screens shared by the registration workflows.',
+        '1: Enter email address',
+        '2: Accept EULA',
+        '3: Verify email',
+        '4: Create password',
+        '5: Add account details',
+        '6: Success & welcome',
+    ]}
+/>
+
+## Joining an Organization
+
+In addition to account creation, some applications may require the user to join groups and organizations. Joining an organization workflow includes the following variations:
+
+**Request Access**
+
+This occurs when a user has an account but need to access an invitation-only organization. The workflow involves the following screens:
+
+-   Enter organization code
+-   Success & welcome (awaiting approval)
+-   Receive an email invitation
+-   Success & welcome
```

Comment:
```suggestion
-   Enter organization code
-   Request success
-   Joined success / welcome
```

Reply:
Should we rename all these Success / Welcome screens to: 

Success: request sent
Success: joined organization
Success: Account created

etc?

With a prefix, it is easier to tell which screens fall in that "success/welcome" category.

Reply:
And same as before, I think email invitation is an integrated part of this experience.

Reply:
Yeah, I think the prefix would work.

I agree sending the email is a critical part of the flow, but it's not a screen. If you want to keep it in there, you could reword the introductory sentence to be something more like "The X workflow involves the following _steps_:"